cli/src/info.rs

```diff
@@ -165,6 +170,10 @@ fn show_arm_info(interface: &mut dyn ArmProbeInterface) -> Result<()> {
     let dp = DpAddress::Default;
     let num_access_ports = interface.num_access_ports(dp).unwrap();
 
+    if num_access_ports == 0 {
+        tree.push("No memory APs found. The chip has locked debug access.".to_string());
```

Comment:
The memory access ports can still be found if they're locked, it's just not possible to access the memory.